patch-setup-config.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Yes, as long as I found a nicer way to handle binary patching in Haskell, I can just use the `Version` property of `Cabal` package in the Haskell space I guess.